dscreator/datasets/trajectories/ferrybox.py

```diff
@@ -171,3 +171,68 @@ def dataset_attributes(self, ds: xr.Dataset) -> FerryboxDatasetAttrs:
         )
         attrs.project += "," + ",".join(["AquaINFRA"])
         return attrs
+
+
+@dataclass
+class RamsesFantasy(TrajectoryDatasetBuilder):
+    def dataset_attributes(self, ds: xr.Dataset) -> FerryboxDatasetAttrs:
+        return FerryboxDatasetAttrs(
+            title="RAMSES on MS Color Fantasy",
+            title_no="RAMSES på MS Color Fantasy",
+            summary="RAMSES on MS Color Fantasy. Also see: https://www.niva.no/en/ferrybox & https://www.colorline.no/oslo-kiel/fakta.",
```

Comment:
I'd provide a bit more info here, something like "Ocean surface reflectance derived from spectral imaging radiometer (RAMSES) measurements on board of a ferry sailing from Oslo, Norway to Kiel, Germany. For more information see ..."